modules/shBidAdapter.js

```diff
@@ -57,6 +68,7 @@ export const spec = {
         type: streamType,
         bidId: bid.bidId,
         mediaType: isBanner ? BANNER : VIDEO,
```

Comment:
out of curiosity:
will video ads still be displayed if the media type is banner _and_  video?

Reply:
Our player will work and show video or display ad as banner in an iframe, in this case.